node/src/actors/storage_keys.rs

```diff
@@ -3,3 +3,6 @@ pub static PEERS_KEY: &'static [u8] = b"peers";
 
 /// Constant to specify the chain state key for the storage
 pub static CHAIN_STATE_KEY: &'static [u8] = b"chain";
+
+/// Constant to specify the secret key key for the storage
+pub static EXTENDED_SK_KEY: &'static [u8] = b"secret";
```

Comment:
This is an extended secret key, but not just any extended secret key. Let's clearly identify the master key as such.
```suggestion
pub static MASTER_KEY: &'static [u8] = b"master_key";
```